test/CalculatorTest.php

```diff
@@ -111,4 +111,10 @@ public function testDivisionNegative(){
         $this->assertNotEquals($returnValue,5);
     }
     
+    public function testDivisionByZero(){
+        
+        $this->expectException(CalculatorException::class);
+        $this->calculator->division(0,2);
+    }
+    
```

Comment:
la validacio hauria de ser al reves (2/0)
i estaria be afegir la validacio de (0/2) == 0